app/docs/0.11.x/configuration.md

```diff
@@ -589,99 +582,62 @@ name                            | description
-same cluster.
-
-Kong's clustering works on the IP layer (hostnames are not supported, only
-IPs) and expects a flat network topology without any NAT between the
-datacenters.
-
-A common pattern is to create a VPN between the two datacenters such that
-the flat network assumption is not violated.
-
-See the clustering reference for more informations:
-https://getkong.org/docs/latest/clustering/
-
----
-
-##### **cluster_listen**
-
-Address and port used to communicate with other nodes in the cluster.
-All other Kong nodes in the same cluster must be able to communicate over both
-TCP and UDP on this address. Only IPv4 addresses are supported.
-
-Default: `0.0.0.0:7946`
-
----
-
-##### **cluster_listen_rpc**
-
-Address and port used to communicate with the cluster through the agent
-running on this node. Only contains TCP traffic local to this node.
-
-Default: `127.0.0.1:7373`
-
----
-
-##### **cluster_advertise**
-
-By default, the `cluster_listen` address is advertised over the cluster.
-If the `cluster_listen` host is '0.0.0.0', then the first local, non-loopback
-IPv4 address will be advertised to other nodes.
-However, in some cases (specifically NAT traversal), there may be a routable
-address that cannot be bound to. This flag enables advertising a different
-address to support this.
-
-Default: none
-
----
-
-##### **cluster_encrypt_key**
+#### Datastore cache section
 
-Base64-encoded 16-bytes key to encrypt cluster traffic with.
+In order to avoid unecessary communication with the datastore, Kong caches
+entities (such as APIs, Consumers, Credentials...) for a configurable period
+of time. It also handles invalidations if such an entity is updated.
 
-Default: none
+This section allows for configuring the behavior of Kong regarding the
+caching of such configuration entities.
 
 ---
 
-##### **cluster_keyring_file**
+##### **db_update_frequency**
 
-Specifies a file to load keyring data from.
-Kong is able to keep encryption keys in sync
-and perform key rotations. During a key
-rotation, there may be some period of time in
-which Kong is required to maintain more than
-one encryption key until all members have
-received the new key.
+Frequency (in seconds) at which to check for
+updated entities with the datastore.
+When a node creates, updates, or deletes an
+entity via the Admin API, other nodes need
+to wait for the next poll (configured by
+this value) to eventually purge the old
+cached entity and start using the new one.
 
-Default: none
+Default: 5 seconds
 
 ---
 
-##### **cluster_ttl_on_failure**
-
-Time to live (in seconds) of a node in the cluster when it stops sending
-healthcheck pings, possibly caused by a node or network failure.
-If a node is not able to send a new healthcheck ping before the expiration,
-other nodes in the cluster will stop attempting to connect to it.
+##### **db_update_propagation**
 
-Recommended to be at least `60`.
+Time (in seconds) taken for an entity in the
```

Comment:
because they are a 1-on-1 copy of the configuration file, same reason for not addressing the other ones, as they are from that file.